### PR TITLE
Complete implementation of a Boolean Column.

### PR DESCRIPTION
**Pull request by vivekmenezes**

### CODE REVIEW

File: sql/server.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ func (p parameters) Arg(i int) (parser.Datum, bool) {
 	case *string:
 		return parser.DString(*t), true
 	default:
-		panic(fmt.Sprintf("Incorrect type %T", t))
+		panic(fmt.Sprintf("Unexpected type %T", t))
```
Comment (petermattis) [inline sql/server.go]: Generally error messages are uncapitalized. I extend this to panics as well. `s/Unexpected/unexpected/g`

File: sql/table.go [hunk 2]
```diff
@@ -80,6 +82,8 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 				col.Type.Kind = structured.ColumnType_TEXT
 			case *parser.BlobType:
 				col.Type.Kind = structured.ColumnType_BLOB
+			default:
+				panic(fmt.Sprintf("Unexpected type %T", t))
```
Comment (petermattis) [inline sql/table.go]: `s/Unexpected/unexpected/g`

**Comment by petermattis** [comment]
LGTM

**Comment by vivekmenezes** [comment]
Incorporated the two suggested changes to the error messages. Thanks.

File: sql/server.go [hunk 3]
```diff
@@ -145,7 +145,7 @@ func (p parameters) Arg(i int) (parser.Datum, bool) {
 	case *string:
 		return parser.DString(*t), true
 	default:
-		panic(fmt.Sprintf("Incorrect type %T", t))
+		panic(fmt.Sprintf("unexpected type %T", t))
```
Comment (tamird) [inline sql/server.go]: did you forget to push before merging?

Comment (tamird) [inline sql/server.go]: oh, disregard; the comment was to _not_ capitalize

**Pull request merged**